packages/kotti-ui/source/next/kotti-field-date/styles.scss

```diff
@@ -159,6 +159,38 @@ $year-font-size: 1rem;
 				}
 			}
 		}
+
+		.in-range {
+			background-color: var(--interactive-04);
+
+			@supports (clip-path: inset(15% -50% 15% -50%)) {
+				clip-path: inset(15% -50% 15% -50%);
+			}
+
+			&.start-date {
+				background: linear-gradient(
+					270deg,
+					var(--interactive-04) 0%,
+					var(--interactive-04) 50%,
+					rgba(0, 0, 0, 0) 50%
+				);
+			}
+			&.end-date {
+				background: linear-gradient(
+					90deg,
+					var(--interactive-04) 0%,
+					var(--interactive-04) 50%,
+					rgba(0, 0, 0, 0) 50%
+				);
+			}
+			&.start-date.end-date {
+				background: transparent !important;
```

Comment:
please no use of !important
elaborate on what was happening that led u to do that?

Reply:
so the fix for this, is suggested below (exclude .in-range) from td:hover

and it will automatically include start-date and end-date

if you look at the added classes by element-ui, you'll see why

Reply:
c5711aec7d881cceccac92a53db555db7d098edd